bjoern.py

```diff
@@ -35,11 +35,13 @@ def bind_and_listen(host, port=None, reuse_port=False,
     return sock
 
 
-def server_run(sock, wsgi_app):
-    _bjoern.server_run(sock, wsgi_app)
+def server_run(sock, wsgi_app, statsd=None):
+    if statsd is None:
+        _bjoern.server_run(sock, wsgi_app)
```

Comment:
this will break when compiled with statsd and statsd options are not provided.

Reply:
Yes, unfortunately in the current implementation we can't know whether bjoern is compiled with statsd support. Maybe we should simply add an attribute that we can check.

Reply:
Sure, I think we should add attributes for all features that can be consulted from python and that way people will be able to write code that can selectively enable or disable a feature.

something like 

 - `bjoern.has_statsd`
 - `bjoern.has_statsd_tags`
 - `bjoern.has_signal_handling`

and so on.

Reply:
Sounds good. Maybe we can use a single attribute "features" or "compilation flags" or whatever, and make it a list that we simply populate as follows

```
l = PyList_New();
#ifdef WANT_X
PyList_Append(l, PyString_FromString("X"));
#endif 
...
```